### ISSUE TITLE
VictoryVoronoiContainer fires onActivated event with invalid points

### ISSUE THREAD

**Issue by yangdong** [FormidableLabs/victory-native#242]
![voronoi](https://user-images.githubusercontent.com/954058/35027713-30fd8f98-fb8d-11e7-8164-cc9f58b0e751.gif)
<img width="522" alt="victoryvoronoicontainer" src="https://user-images.githubusercontent.com/954058/35027715-3155cdde-fb8d-11e7-83b7-76874344503a.png">

Here is my code:

```
import React, { Component } from 'react';
import Section from '../components/Section';
import TitleAndValue from '../components/TitleAndValue';
import { getCrafts } from '../actions/craft';
import { connect } from 'react-redux';
import { bindActionCreators } from 'redux';
import { HeaderBar } from '../components/HeaderBar';
import { Container, Text } from "native-base";

import {View} from 'react-native';

import themeVar from '../theme/variables/common';

import { VictoryLine,VictoryTheme,VictoryChart, VictoryGroup, VicotryGroup,Line,VictoryVoronoiContainer,Curve, VictoryLabel, VictoryBar,VictoryAxis,VictoryScatter } from 'victory-native';

export class IndicatorChart extends React.Component {

  state = {
    x: null,
    y: null,
    cursorActivated: false
  }

  activeCursor(points, props) {
    this.setState({
        cursorActivated: true,
        x: points[0].x,
        y: points[0].y,
    })
  }

  render() {
    const {domain, data} = this.props;

    console.warn(this.state)

    return (
      <VictoryChart theme={VictoryTheme.material} domain={domain} scale={{y: "linear", x: "time"}}
        containerComponent={ <VictoryVoronoiContainer voronoiDimension="x" onActivated={(points, props) => this.activeCursor(points, props)}/>}>

        {
          this.state.cursorActivated && <VictoryGroup>
          <VictoryLine style={{ data: { stroke: "black", strokeWidth: 0.5 },}} data={[{ x: domain.x[0], y: this.state.y }, { x: domain.x[1], y: this.state.y },]}/>
          <VictoryLine style={{ data: { stroke: "black", strokeWidth: 0.5 },}} data={[ { x: this.state.x, y: domain.y[0]}, { x: this.state.x, y: domain.y[1]},]}/>
          </VictoryGroup>
        }

        <VictoryLine name={'line'} data={data} interpolation={'monotoneX'}
          style={{
            data: {
              stroke: "#c43a31", strokeWidth: 1
            },
            labels: {
              fontSize: 15,
              fill: (d) => "#c43a31"
            }
          }}
        />

        <VictoryAxis tickCount={5} tickFormat={(t) => `${t.getUTCHours()}`}/>
        <VictoryAxis dependentAxis tickCount={5}/>
      </VictoryChart>);
  }
}

```

**Comment by reubenab** [comment]
I think this is an issue with `voronoiDimension={'x'}`. If you don't use that, then it works fine - or at least I ran into a similar issue and that was the problem. Of course, would like that bug fixed too.

**Comment by boygirl** [comment]
this should be fixed in the latest version `0.18.0`
